src/legacy/core_plugins/interpreter/public/functions/kibana.ts

```diff
@@ -18,27 +18,40 @@
  */
 
 import { i18n } from '@kbn/i18n';
+import { ExpressionFunction, KibanaContext } from '../../types';
 
-export const kibana = () => ({
+export type ExpressionFunctionKibana = ExpressionFunction<
+  'kibana',
+  KibanaContext | null,
+  object,
+  KibanaContext
+>;
+
+export const kibana = (): ExpressionFunctionKibana => ({
   name: 'kibana',
   type: 'kibana_context',
-  context: {},
+
+  context: {} as any,
+  // context: {
+  // types: ['kibana_context', 'null'],
+  // },
```

Comment:
I'm not sure what the correct value here should be. The commented out value breaks visualizations.

Reply:
what error are you getting ? currently visualizations don't use the new service, maybe they don't set the type on the input context ?

Reply:
This is what I see when using the other `context` definition:

![image](https://user-images.githubusercontent.com/9773803/66295346-5d319580-e8eb-11e9-876c-908fed621778.png)

I does not print anything to console, but all visualizations break with some error alert.

Reply:
Fixed above error.